libs/cardano-ledger-test/src/Test/Cardano/Ledger/Generic/Trace.hs

```diff
@@ -224,26 +247,38 @@ raiseMockError ::
   String
 raiseMockError slot (SlotNo next) epochstate pdfs txs GenState {..} =
   let utxo = unUTxO $ (_utxo . lsUTxOState . esLState) epochstate
+      _poolParams = (_pParams . dpsPState . lsDPState . esLState) epochstate
    in show $
         vsep
-          [ pcSmallUTxO reify utxo txs,
+          [ ppString "===================================",
+            ppString "UTxO\n" <> pcSmallUTxO reify utxo txs,
+            ppString "===================================",
+            ppString "Stable Pools\n" <> ppSet pcKeyHash gsStablePools,
             ppString "===================================",
-            ppString "Stable Pools\n" <> ppSet ppKeyHash gsStablePools,
+            -- ppString "Initial Pool Distr\n" <> ppMap pcKeyHash pcIndividualPoolStake gsInitialPoolDistr,
+            -- ppString "===================================",
+            -- ppString "Initial Pool Params\n" <> ppMap pcKeyHash pcPoolParams gsInitialPoolParams,
```

Comment:
Dead code?
```suggestion
```

Reply:
When you are trying to debug a failure, its is never clear what things are helpful, and what things are a distraction, so I left these commented out since the last thing I debugged these were distractions. But they were very helpfull in other things. I will leave a comment about this.